include/knowrob/integration/python/converter/dict.h

```diff
@@ -10,7 +10,7 @@
 namespace knowrob::py {
 
 	// Function to convert Python object to boost::any
-	boost::any python_to_boost_any(const boost::python::object &obj) {
+	inline boost::any python_to_boost_any(const boost::python::object &obj) {
```

Comment:
Is there a strong reason for inlining this functions? I mean, shouldn't hurt, just wondering why you did change that

Reply:
Yes it hurts, when I build static libraries, it's expected to have a library as the implementation of a header file. In this case they are the same so it wouldn't work.

Reply:
ok fine